tests/acceptance/pageObjects/FilesPageElement/fileActionsMenu.js

```diff
@@ -139,6 +140,12 @@ module.exports = {
      */
     move: function(target) {
```

Comment:
Just seeing that the `target` param is not used in the function body, so you could remove it.

Reply:
Must have overlooked this. Nice catch!